frontend/elements/src/entry/home/sections/jiggling-section.ts

```diff
@@ -9,15 +10,21 @@ export class _ extends LitElement {
      
     }
    main{
-     background-image: url("Strip_Background_Jigglings@2x.jpg");
-     
+     width:1920px;
+     height:890px;
+     background-image: url("path/to/Strip_Background_Jigglings@2x.jpg");
```

Comment:
??

Reply:
Would need your assistance how to put the background image. It didnt work to do it the way you suggested.

Reply:
ok, we'll merge as-is - but please open a separate issue and assign both of us so it doesn't get lost by accident